draft-ietf-tls-tls13.md

```diff
@@ -4536,7 +4537,8 @@ New cipher suite values are assigned by IANA as described in
 # Implementation Notes
 
 The TLS protocol cannot prevent many common security mistakes. This section
-provides several recommendations to assist implementors.
+provides several recommendations to assist implementors. Also
```

Comment:
minor nit: should be a comma after "Also", here

Alternatively, just drop the "Also" entirely; it's unneeded.